pkg/api/models/kluster_spec.go

```diff
@@ -187,6 +198,23 @@ func (m *KlusterSpec) validateAudit(formats strfmt.Registry) error {
 	return nil
 }
 
+func (m *KlusterSpec) validateAuthenticationConfiguration(formats strfmt.Registry) error {
```

Comment:
Would it make sense to check for the minimum kluster version here as well?

Reply:
That's hard to do. The kluster_spec.go is generated and I don't know of a way to add custom validation at this point.